templates/default.conf

```diff
@@ -23,21 +27,23 @@ include /etc/nginx/conf.d/custom/*.conf;
 
 server {
     server_name ${NGINX_DEFAULT_SERVER_NAME};
-    # Please note that nginx always has a default server. 
-    # In the absence of any server block explicitly marked 
-    # as default_server , nginx will use the first server 
+    # Please note that nginx always has a default server.
+    # In the absence of any server block explicitly marked
+    # as default_server , nginx will use the first server
     # with a matching listen directive.
     # If defaul_server is set, this server will be the
     # default, catch all server, regardless of server_name.
     listen ${NGINX_DEFAULT_SERVER_PORT} ${DEFAULT_SERVER};
 
     #hstsheader
+    if ($new_scheme_host_and_uri) {
+        return 301 $new_domain_uri;
+    }
     if ($new_host_and_uri) {
-        return 301 $new_host_and_uri;
+        return 301 $new_domain_uri;
```

Comment:
I'm looking at the work done by @Monska85 in https://github.com/sparkfabrik/docker-php-drupal-nginx/pull/81/files and wondering if it's ok to change this line (43) and line 46 below

Reply:
Also I think we should update the Readme+Changelog like we did in #81